src/posts/react-image-gallery.md

```diff
@@ -1,33 +1,31 @@
 ---
 title: 'How to use React Image Gallery to build a beautiful and responsive image gallery'
```

Comment:
Some files are too big for the Web like 478 kb and 1.3 mb. Use photopea.com to make them smaller.

Reply:
Already converted to jpg and reduced size !